fanuc_r2000ic_support/launch/test_r2000ic125l.launch

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<launch>
+  <include file="$(find fanuc_r2000ic_support)/launch/load_r2000ic125l.launch" />
+  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
+    <param name="use_gui" value="true" />
+  </node>
```

Comment:
```suggestion
  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui" />
```

because of:

```
<testsuite errors="0" failures="1" name="roslaunch-check_tests_roslaunch_test_r2000ic125l.xml.xml" tests="1" time="1"><testcase classname="roslaunch.RoslaunchCheck" name="fanuc_r2000ic_support_tests_roslaunch_test_r2000ic125l_xml" status="run" time="1"><failure message="roslaunch check [/root/target_ws/src/fanuc_experimental/fanuc_r2000ic_support/tests/roslaunch_test_r2000ic125l.xml] failed" type="" /></testcase><system-out>&lt;![CDATA[
[/root/target_ws/src/fanuc_experimental/fanuc_r2000ic_support/tests/roslaunch_test_r2000ic125l.xml]:
	Missing package dependencies: fanuc_r2000ic_support/package.xml: joint_state_publisher
]]&gt;</system-out></testsuite>
-------------------------------------------------
build/fanuc_r2000ic_support/test_results/fanuc_r2000ic_support/roslaunch-check_tests_roslaunch_test_r2000ic125l.xml.xml: 1 tests, 0 errors, 1 failures, 0 skipped
Summary: 14 tests, 0 errors, 1 failures, 0 skipped
```